mbuild/utils/conversion.py

```diff
@@ -52,21 +59,20 @@ def RB_to_OPLS(
 
     .. warning:: The :math:`\frac{f_{0}}{2}` term is the constant for the OPLS dihedral equation
         and is only used to test... will be shifted by the :math:`\frac{f_{0}}{2}`
```

Comment:
Check line 61, I think this part of the sentence should not have an ellipses.

Reply:
Changed too

    
    .. warning:: The :math:`\frac{f_{0}}{2}` term is the constant for the OPLS dihedral equation.
        If the f0 term is not zero, the dihedral is not an exact conversion;
        since this constant does not contribute to the force equation,
        this should provide matching results for MD, but the energy for each
        dihedral will be shifted by the :math:`\frac{f_{0}}{2}` value.